code/__DEFINES/DNA.dm

```diff
@@ -92,6 +93,23 @@
 #define DNA_SKIN_TONE_BLOCK 3
 #define DNA_UNI_IDENTITY_BLOCKS 7
 
+#define DNA_FEATURE_BLOCKS 15
+#define DNA_MCOLOR_BLOCK 1
+#define DNA_ETHCOLOR_BLOCK 2
```

Comment:
What are "mcolor" and "ethcolor"?
Don't save on variable name length, aim for clarity.